### PR TITLE
changed profile URL

### LINKED ISSUE #64: Change pic url in member profiles
A lot of member profiles don't use github avatars since we want some kind of uniformity lets change all those profile pic urls that don't use github avatars.

Acceptance criterais:
- All such urls are changed.
- PR will contain only one commit.

Who wants it? @p-society/2021-batch

### PR DESCRIPTION
**Pull request by pranayag15**
Fixes #64

### CODE REVIEW

**Comment by pranayag15** [comment]
@palash25 please review.

File: Gemfile.lock [hunk 1]
```diff
@@ -110,6 +110,7 @@ GEM
       thread_safe (~> 0.1)
     tzinfo-data (1.2018.5)
       tzinfo (>= 1.0.0)
+    wdm (0.1.1)
```
Comment (palash25) [inline Gemfile.lock]: Why the change in dependencies? Do we need these @knrt10 ?

File: .bashrc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Ruby exports
```
Comment (knrt10) [inline .bashrc]: @pranayag15 why did you create this file?

File: Gemfile.lock (hunk 1 above)
Comment (knrt10) [reply Gemfile.lock]: Please changes it back to what it was.

File: .bashrc (hunk 2 above)
Comment (palash25) [reply .bashrc]: We don't need this file please delete this.

Comment (pranayag15) [reply .bashrc]: I don't know how it got installed but I will change it back to what it was.

**Comment by palash25** [comment]
@pranayag15 please amend your commits so that we can merge this soon. This is blocking another issue.

**Comment by pranayag15** [comment]
@palash25   I guess I have to reinstall the packages to their older version,  I'll definitely do it today.

**Comment by palash25** [comment]
>  I guess I have to reinstall the packages to their older version, I'll definitely do it today.

No just change the version to the way they were before and delete the files that were not supposed to be there. Then amend and force push and let the tests run on the CI. If it passes then we will merge.

No need to reinstall packages, broken toolchains can take a little time to fix and we need to merge this.

**Comment by pranayag15** [comment]
@knrt10 have changed accordingly. plz review
